packages/page-alerts/src/sass/_module.scss

```diff
@@ -0,0 +1,121 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// page-alerts module
+// Use page alerts to notify users of important information and state changes to the page.
+//
+// Content:
+// - Generic page alert styling
+// - Generic page alert icon position styling
+// - Page alert error
+// - Page alert warning
+// - Page alert success
+// - Page alert info
+// - Page alert screen-reader only
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import '_globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Page alert styling
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Generic page alert styling.
+ */
+.uikit-page-alerts {
+	position: relative;
+	margin: uikit-space( 1 ) 0;
+	padding: uikit-space( 0.75 ) uikit-space( 0.75 ) uikit-space( 0.5 ) 72px;
+	border: solid 2px $uikit-colour-Border;
```

Comment:
We should default to --info when no modifier is set and not introduce yet another variation.